main.js

```diff
@@ -1,25 +1,32 @@
 var level = require("level");
 var db  = level("./student-db", { valueEncoding: 'json'});
-
+var status = ['Applying', 'Under Interview', 'Exam Pending', 'Admitted', 'Probationary'];
 
 const prompt = require('prompt-sync')({sigint: true});
 
  var stId = prompt('Enter ID: ');
  var fullName = prompt('Enter full Name: ');
- var stdAge = prompt('enter age: ');
+ var stdAge = prompt('Enter age: ');
  var stdAddress = prompt('Enter address: ');
 
 acceptStudents(stId, fullName,stdAge, stdAddress);
 
-function acceptStudents(stId, fullName,stdAge){
-    db.put('student', {id: Number(stId) ,name: fullName, age: Number(stdAge), address: stdAddress, status: 'applying' }, function(err){  
+async function acceptStudents(stId, fullName,stdAge){
+    await db.put('student', {id: Number(stId) ,name: fullName, age: Number(stdAge), address: stdAddress}, function(err){  
         output();
+        scheduleInterview(stId, 'February 26, 2021');
     })
-
 }
 
-function output(){
-    db.get('student', function(err, student){
-        console.log(student);
+async function output(){
+    await db.get('student', function(err, student){
+        console.log(student);     
+        console.log('Status: ' + status[0]);
+    })
+    }
+async function scheduleInterview(stId, scheduleDate){
+    await db.get(stId, function(err, value){
+        console.log('\nInterview date: ' + scheduleDate);
+        console.log('Status: ' + status[1]);
```

Comment:
You tried to get  your student but did not check the value of err
To set the schedule you need to call `db.put(stdId` ..... after `db.get(stdId`.....